seed/static/seed/partials/data_upload_modal.html

```diff
@@ -30,50 +30,113 @@ <h4 class="modal-title" ng-switch-when="16" translate>PM_METER_IMPORT_RESULTS</h
         </div>
     </div>
     <!-- Step 2: Choose upload type -->
```

Comment:
Maybe we should update this comment now that this handles step `2` and step `13`